modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/BaseCKEditorConfigContributor.java

```diff
@@ -40,12 +44,28 @@ public void populateConfigJSONObject(
 
 		jsonObject.put("allowedContent", Boolean.TRUE);
 
+		StringBundler sb = new StringBundler(5);
+
+		sb.append("cke_editable html-editor");
```

Comment:
It's great to see it fixed but I'm a bit worried about modifying a CKEditor default behavior. Would be good document more a bit this part. Could you bring this description

```
The root cause of the issue is that CKEditor sets the cke_editable class through JS, after the initial render. This results in FOUC on the first setting of data, and, far more noticeably, in the case of content sync in Forms. To fix this, we are setting the cke_editable class in configuration, through config.bodyClass.
```

to the COMMIT_MSG?

Reply:
I though about this, but I believe it is safe to do. Balloon editor has the same editable area, and so do BBCode and Creole versions of Classic Editor. All of them share the same implementation in [editable.js](https://github.com/ckeditor/ckeditor4/blob/4.16.1/core/editable.js), meaning the same CSS class in body.